Update requirements for early disclosure list (#664)
istio/community: EARLY-DISCLOSURE.md

4. Be a participant and active contributor in the Istio community.
5. Accept the [Embargo Policy](#embargo-policy) that is outlined above.

#### Additional membership criteria (Feb 14, 2022)

> As of Februrary 14, 2022 there is additional criteria to be a member of the early disclosure list.
> 
> 6. Be a member of the [Envoy Early Disclosure list](https://github.com/envoyproxy/envoy/blob/main/SECURITY.md#members).
> 7. Have a member of your organization be a participant in the Istio Product Security Working Group
>   a. This is a private meeting held alternating Wednesdays at 11am Pacific or 3pm Pacific to accommodate a world wide community.

Reviewer: ```suggestion
As of February 14, 2022 there are additional criteria to be a member of the early disclosure list:

6. Be a member of the [Envoy early disclosure list](https://github.com/envoyproxy/envoy/blob/main/SECURITY.md#members).
7. Have a member of your organization be a participant in the Istio Product Security Working Group. This is a private meeting held alternating Wednesdays at 11am Pacific or 3pm Pacific to accommodate a world-wide community.
```

> 5. Accept the Embargo Policy that is outlined above.

We accept.

> 6. Be a member of the [Envoy Early Disclosure list](https://github.com/envoyproxy/envoy/blob/main/SECURITY.md#members).

> 
> We are listed in the Envoy early disclosure membership program.
> > 7. Have a member of your organization be a participant in the Istio Product Security Working Group

Reviewer: ```suggestion

We are listed in the Envoy early disclosure membership program.

> 7. Have a member of your organization be a participant in the Istio Product Security Working Group
```

Reviewer: sorry - just one missed and then we're good to go

> 5. Accept the Embargo Policy that is outlined above.

We accept.

> 6. Be a member of the [Envoy Early Disclosure list](https://github.com/envoyproxy/envoy/blob/main/SECURITY.md#members).

We are listed in the Envoy early disclosure membership program.

> 
> 

Reviewer: ```suggestion

```
